src/sdk/encrypt.ts

```diff
@@ -20,15 +21,17 @@ export type ZKInput = {
   add32: (value: number | bigint) => ZKInput;
   add64: (value: number | bigint) => ZKInput;
   add128: (value: number | bigint) => ZKInput;
+  add256: (value: number | bigint) => ZKInput;
+  addBytes64: (value: Uint8Array) => ZKInput;
+  addBytes128: (value: Uint8Array) => ZKInput;
+  addBytes256: (value: Uint8Array) => ZKInput;
   addAddress: (value: string) => ZKInput;
-  getValues: () => bigint[];
   getBits: () => number[];
-  resetValues: () => ZKInput;
   encrypt: () => {
```

Comment:
can you make this function async please? this is needed to get parity with mocked mode, and I believe it makes more sense anyways wdyt? @immortal-tofu

Reply:
Yes it will be the case when I'll add fetch() to the gateway, don't worry